asterisk/agi/templates/faxmail/default/en/body

```diff
@@ -0,0 +1,11 @@
+Greetings!
```

Comment:
We could do a new issue to replace current template model to Zend_View and standarize them everwhere. We're also using two files (subject and body) which I don't like too much (maybe using the first line of the template for subject, rest for body?).

This basic _replace plain/text_ mode is used for:
 - [Voicemail notifications](https://github.com/irontec/ivozprovider/blob/bleeding/asterisk/agi/application/controllers/VoicemailController.php#L55)
 - [Recordings Disk Usage](https://github.com/irontec/ivozprovider/blob/bleeding/kamailio/recordings/application/controllers/RecordingsController.php#L353)
 - And now Incoming Faxes

I prefer to start with all of them in the same way and change them afterwards in the same change to avoid multiple implementations for the same thing.

What do you think?